lib/yardstick/v2_client/user.rb

```diff
@@ -7,6 +7,12 @@ class User
       include RemoteModel
       include Cacheable
 
+      class Paths < Yardstick::ActiveModel::Base
+        attr_accessor :grants
+      end
+
+      resource_uri '/v2/users'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.